sovtoken/sovtoken/request_handlers/token_utils.py

```diff
@@ -85,3 +86,15 @@ def validate_given_inputs_outputs(inputs_sum, outputs_sum, required_amount, requ
     raise InvalidClientMessageException(getattr(request, f.IDENTIFIER.nm, None),
                                         getattr(request, f.REQ_ID.nm, None),
                                         'Request to not meet minimum requirements')
+
+
+def commit_to_utxo_cache(utxo_cache, state_root):
```

Comment:
I think, we should create a UTXOCacheBatchHandler for this and register it into write_manager (as for idr_cache)

Reply:
ok

Reply:
Maybe it should be removed because it dublicated in UTXOBatchHandler?